Bottom: e69aa23a5fdd3ee629d0de87b0ef4ef0e6e750ad
Top:    1d7095261984b9ac5f5a965bdb7d5da3e4ccd343
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-24 21:19:03 -0400

Refresh of of-soc-simple-rework

---

diff --git a/sound/soc/fsl/Kconfig b/sound/soc/fsl/Kconfig
index dc79bdf..8060b95 100644
--- a/sound/soc/fsl/Kconfig
+++ b/sound/soc/fsl/Kconfig
@@ -1,5 +1,5 @@
 config SND_SOC_OF_SIMPLE
-	tristate
+	bool
 	
 config SND_MPC52xx_DMA
 	tristate
